tests/test_loadReferenceCatalog.py

```diff
@@ -63,6 +63,14 @@ def loadSkyCircle(self, ctrCoord, radius, filterName, **kwargs):
     def loadPixelBox(self, bbox, wcs, referenceFilter, **kwargs):
         return self.loadSkyCircle(None, None, referenceFilter)
 
+    def loadSchema(self, filterName):
```

Comment:
When does this get used within this test suite?
I guess what I'm asking is - why is it needed (not a criticism, just a query)?

Reply:
The base class implementation of this method that I just added in `meas_algorithms` uses `self.refCats`, which is set to an empty list by this testing subclass.  So I needed to override it here to do something else.

It's used whenever `LoadReferenceCatalogTask._determineFluxFields` is called.